packages/tpetra/core/src/Tpetra_CrsGraph_def.hpp

```diff
@@ -749,13 +749,16 @@ namespace Tpetra {
     , fillComplete_(false)
     , lowerTriangular_ (false)
     , upperTriangular_ (false)
-    , indicesAreSorted_(true)
     , noRedundancies_(true)
     , haveLocalConstants_ (false)
     , haveGlobalConstants_ (false)
     , sortGhostsAssociatedWithEachProcessor_(true)
   {
     staticAssertions ();
+    if(!params.is_null() && !params->get<bool>("sorted"))
```

Comment:
If `params` is not null but "sorted" isn't in it, then one-argument `get` will throw.  (Two-argument `get` sets a default value if the parameter doesn't exist.)  You could try this instead:
```c++
if (! params.is_null () && params->isParameter("sorted") &&
  ! params->get<bool>("sorted")) // etc.
```